tests/unit_tests/pyanaconda_tests/modules/storage/test_module_device_tree.py

```diff
@@ -341,6 +342,23 @@ def test_get_zfcp_device_data(self):
             "path-id": "ccw-0.0.010a-fc-0x5005076300c18154-lun-0x5719000000000000"
         })
 
+    def test_get_parted_device_data(self):
+        sda1 = PartitionDevice(
+            "dev1",
+            size=Size("500 MiB"),
+            fmt=get_format("ext4", exists=True)
+        )
+        sda1._parted_partition = Mock(**{'type': PARTITION_NORMAL,
+                                         'getLength.return_value': int(sda1.size),
+                                         'getFlag.return_value': 0,
+                                         'name': "Microsoft reserved partition",
+                                         'number': 1})
+
+        self._add_device(sda1)
+
+        data = self.interface.GetDeviceData("dev1")
+        # FIXME: assert data['parted-partition-name'] is Microsoft.. 
```

Comment:
`name` is an argument for the Mock constructor so you need to use `configure_mock` to set it after creating the `Mock` instance, see https://python.readthedocs.io/en/latest/library/unittest.mock.html#mock-names-and-the-name-attribute

Reply:
Thank you